tests/common/fixtures/advanced_reboot.py

```diff
@@ -39,6 +39,16 @@ def __init__(self, request, duthost, ptfhost, localhost, tbinfo, creds, **kwargs
             "Please set rebootType var."
         )
 
+        if duthost.facts['asic_type'] == 'vs':
+            self.kvmTest = True
+            # skip testcase if it is not marked as kvm compatible
+            # TODO: A better way could be selecting kvm testcases with the vs device type marker
+            #       Remove this check once the vs testcases are selected using the marker
+            if 'kvmCompatible' not in kwargs or kwargs['kvmCompatible'] != True:
```

Comment:
>kvmCompatible [](start = 56, length = 13)

Why we need this?

Reply:
Removed this and checking pytest mark instead.